### PR TITLE
Add post functions

### PR DESCRIPTION
**Pull request by aibaars**

### CODE REVIEW

File: src/main/java/com/semmle/jira/addon/workflow/LgtmDismissAlert.java [hunk 1]
```diff
@@ -32,6 +32,7 @@
  */
 public class LgtmDismissAlert extends AbstractJiraFunctionProvider {
   public static final String FIELD_TRANSITION = "transitionField";
+  static final String FULL_MODULE_KEY = "com.semmle.lgtm-jira-addonlgtm-dismiss-alert";
```
Comment (nickfyson) [inline src/main/java/com/semmle/jira/addon/workflow/LgtmDismissAlert.java]: Was this supposed to have another `.` in it?

**Comment by nickfyson** [comment]
This seems to be working nicely! I was tripped up at first by not realising the attaching of post functions happened when the config was saved, but of course customers won't have the webhook URL present at all unless they've reconfigured the plugin...)

File: src/main/java/com/semmle/jira/addon/workflow/LgtmDismissAlert.java (hunk 1 above)
Comment (aibaars) [reply src/main/java/com/semmle/jira/addon/workflow/LgtmDismissAlert.java]: No, I don't think so.

Jira seems to simply join the plugin ID and the function ID. See for example:
```
<arg name="full.module.key">
com.atlassian.jira.plugin.system.workflowupdate-issue-field-function
</arg>
```

**Comment by aibaars** [comment]
This has been simplified. No code changes required anymore ;-)

@Daverlo @nickfyson

**Pull request merged**